db/migrate/20210504153704_archive_billing_tables.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Archive the billing tables to env_sequencescape_archive
+class ArchiveBillingTables < ActiveRecord::Migration[5.2]
+  include MigrationExtensions::DbTableArchiver
+
+  def change
+    check_archive!
+    archive!('billing_items')
+    archive!('billing_products')
+    archive!('billing_product_catalogues')
+  end
```

Comment:
We have to check if we have permissions to create new databases in production with current user.

Reply:
We've archive tables before, and the database did exist then, but someone seems to have dropped it?!

Reply:
We have `GRANT ALL PRIVILEGES ON `sequenc%`.* TO 'psdp_rw'@'%'` though, so should be fine.

Reply:
Table was archived to NFS.

Reply:
We do have the correct permissions, I double checked. Deleted the comment with the actual grants as it included the username. Not really a major concern, but I'll avoid leaking it if I can. (Edit kept the comment history)

Reply:
Ok